### ISSUE TITLE
Oncomplete event for effects

### ISSUE THREAD

**Issue by MossFrog** [effekseer/EffekseerForWebGL#59]
Is there an onComplete() event for effekseer effects? This could be useful since performance on Android devices is very bad even when no effects are being rendered within the scene (The update loop causes performance issues)

**Comment by durswd** [comment]
Which version did you use? I have improved a performance on the android recently.

should onComplete be called after rendered?

**Comment by MossFrog** [comment]
Yes after the effect has completed its render is it possible to obtain an event in order to trigger further code?
I will try a more recent version for the performance issues! Thank you

**Comment by durswd** [comment]
If it is still slow, please tell me your environment in a detail.

**Comment by MossFrog** [comment]
I have not tested performance on the newer versions yet. I was asking if i can create an event callback function for the effekseer effects, such as 
effect.onComplete() => function() {console.log("EFFECT IS FINISHED"}; ?

**Comment by durswd** [comment]
It seem that it cannot implement it and it makes no sense because JavaScript and WebGL runs on a single thread.

**Comment by MossFrog** [comment]
I see, then would it be possible to set an an effect to loop? Such as effect.setLoop(true); so that the effect can keep looping after completion?

**Comment by durswd** [comment]
I don't have a plan to implement setLoop, because you can implement it with EffekseerHandle.exists()
